lib/knapsack/adapters/rspec_adapter.rb

```diff
@@ -48,10 +48,8 @@ def bind_time_offset_warning
       end
 
       def self.test_path(example_group)
-        unless example_group[:turnip]
```

Comment:
Would be good to still have backward compatibility with Turnip 1.x

You could check if Turnip exists and if the version is bellow 2.x then follow old approach.

```ruby
if defined?(Turnip) && Turnip::VERSION.to_i < 2
```

and we need test for both cases.